### DISCUSSION TITLE
🚀 Thinking about: Using templUI to build a modern deployment platform

### DISCUSSION THREAD

**Discussion by axadrn** [templui/templui#15] [Announcements]
I'm thinking about building an open-source deployment platform (think Vercel/Heroku but self-hosted) using Go + templUI ⚡️. This would be a great way to:

- Showcase how to build complex UIs with templUI
- Identify which components need improvements
- Add new components based on real needs
- Show templUI in a production environment 🛠️

Yes, I know Coolify, Dokploy and others exist - but competition is good! And it would be the perfect way to show templUI's capabilities in a real-world application ✨

Would love to get your thoughts and what features you'd find useful! 🤔

[https://x.com/_axzilla/status/1868269325968347496](https://x.com/_axzilla/status/1868269325968347496)

**Comment by sdil** [comment]
I recently stumbled upon a new alternative work in progress: https://github.com/maddalax/dockman built using htmgo

**Reply by axadrn** [reply]
Oh, thanks for that. Nice to see the ecosystem evolving. I guess I shouldn't let that put me off, because competition is actually the best market validation you can have!
